doc/Reference/Markup/StaticAndThemeResources.md

```diff
@@ -0,0 +1,158 @@
+
+Now to use our `Resource<Geometry>`, we can simply reference it similar to our constant string, except that we need to also add it to a `ResourceDictionary` somewhere in the Visual Tree.
+
+```csharp
+public partial class MyPage : Page
+{
+    public MyPage()
+    {
+        this.Resources(r => r.Add(MyResources.MyIcon))
+            .Content(
+                new PathIcon()
+                    .Data(MyResources.MyIcon)
+            );
+    }
+}
+```
+
+Now let's look at the equivalent code of what this is in effect doing:
+
+```csharp
+public partial class MyPage : Page
+{
+    public MyPage()
+    {
+        this.Resources(r => r.Add<Geometry>("MyIcon", "{path string}"))
+            .Content(
+                new PathIcon()
+                    .Data(x => x.StaticResource("MyIcon"))
+            );
+    }
+}
+```
+
+### Getting an Existing Resource
+
+Similarly, we may not want to use the lambda directly. In this case we can again use the `StaticResource` type to get our resource. This can also be helpful for resources that you may not get as often and which may be added for you already to the Resources of a 3rd Party `ResourceDictionary` you have brought in.
+
+```csharp
+new PathIcon()
+    .Data(StaticResource.Get<Geometry>("MyIcon"))
+```
+
+### Strongly Typed Keys
+
+Along this same path we may want to use a resource frequently, but we don't need to handle creating the resource or adding it to the Resource Dictionary. In this case we might want to instead use the `StaticResourceKey<T>` similar to our `Resource<T>`. In this case we would simply update our `MyResources` class like follows:
+
+```csharp
+public static class MyResources
+{
+    public static readonly StaticResourceKey<Geometry> MyIcon = new StaticResourceKey<Geometry>(nameof(MyIcon));
+}
+```
+
+Since we do not need to add it to the Resource Dictionary as it may have come from a Merged Dictionary higher up in the Visual Tree we now might have code that simply looks like:
+
+```csharp
+public partial class MyPage : Page
+{
+    public MyPage()
+    {
+        this.Content(
+            new PathIcon()
+                .Data(MyResources.MyIcon)
+        );
+    }
+}
+```
+
+### Theme Resources
+
+Similar to the Static Resource's, Theme Resources have all of the same basic helpers:
+
+```csharp
+public static class MyResources
+{
+    public static Action<IDependencyPropertyBuilder<Color>> MyThemeColor =>
+        ThemeResource.Get<Color>("MyThemeColor");
+
+    public static readonly ThemeResourceKey<Color> MyOtherColor = 
+        new ThemeResourceKey<Color>("MyOtherColor");
+}
+```
+
+You may notice in the above example, we don't use `ThemeResource.Create`. This is because this is the one area where as you might imagine there is a small difference. To create a ThemeResource we simple would update our code to provide a 2nd value where the first value is used for the Light Theme and the 2nd value is used for the Dark Theme.
```

Comment:
```suggestion
### Theme Resources

Similar to the Static Resource's, Theme Resources have all of the same basic helpers:

```csharp
public static class MyResources
{
    public static Action<IDependencyPropertyBuilder<Color>> MyThemeColor =>
        ThemeResource.Get<Color>("MyThemeColor");

    public static readonly ThemeResourceKey<Color> MyOtherColor = 
        new ThemeResourceKey<Color>("MyOtherColor");
}
```

You may notice in the above example, we don't use `ThemeResource.Create`. This is because this is the one area where as you might imagine there is a small difference. To create a ThemeResource we simply would update our code to provide a 2nd value where the first value is used for the Light Theme and the 2nd value is used for the Dark Theme.
```